pretty_gpx/gpx/augmented_gpx_data.py

```diff
@@ -226,13 +223,11 @@ def find_huts_between_daily_tracks(list_gpx_path: list[str] | list[bytes],
     list_dist_km: list[float] = []
     list_uphill_m: list[float] = []
     for path in list_gpx_path:
-        gpx_track, dist_km, uphill_m = GpxTrack.load(path)
+        gpx_track = GpxTrack.load(path)
         list_gpx_track.append(gpx_track)
```

Comment:
a list comprehension would be better now that we remove the other two :)

Reply:
fixed in the last commit